app/services/irs_attempts_api/tracker.rb

```diff
@@ -24,6 +24,7 @@ def track_event(event_type, metadata = {})
         device_fingerprint: device_fingerprint,
         user_ip_address: request&.remote_ip,
         irs_application_url: sp_request_uri,
+        client_port: CloudFrontRequest.new(request)&.client_port,
```

Comment:
1. `.new` should never return a nil value, we shouldn't need the `&` here
2. the idea is we'd instantiate one `CloudFrontRequest` and read off `tls_version` etc etc as separate fields, right?

Reply:
1. Good catch. I was being a little too paranoid there.
2. Theoretically yes, but we're not currently using any of the other headers. Things like `tls_version` aren't part of our work; I just implemented everything in the infra PR adding these since it seemed weird to add a new class that only knew about one of them, but I didn't really want it to be the `WeirdClientPortParserThing`.

Reply:
I'm a big fan of "YAGNI" --- let's remove all those other methods if we're not using any of them, we can always add them later if we find uses